src/main/webapp/app/teams/teams-status.component.html

```diff
@@ -11,8 +11,8 @@
             <div class="status-meta-text d-flex flex-column justify-content-center align-items-center">
                 <h5 *ngIf="!team.slogan; else sloganWithTitle" class="team-title mb-0">{{team.name}}</h5>
                 <ng-template #sloganWithTitle>
-                    <h5 class="team-slogan mb-0">{{team.slogan}}</h5>
-                    <h6 class="team-title mb-0">{{team.name}}</h6>
+                    <h5 class="team-title mb-0">{{team.name}}</h5>
```

Comment:
See https://github.com/iteratec/slab-dojo/pull/12/files#diff-4a180018841d874d301188b3144d8d5eR42
```suggestion
                    <h5 class="team-title team-title-with-slogan mb-0">{{team.name}}</h5>
```